programs/virtual-curve/src/instructions/initialize_pool/ix_initialize_virtual_pool_with_spl_token.rs

```diff
@@ -177,6 +179,20 @@ pub fn handle_initialize_virtual_pool_with_spl_token<'c: 'info, 'info>(
         initial_base_supply,
     )?;
 
+    // remove mint authority
+    anchor_spl::token_interface::set_authority(
+        CpiContext::new_with_signer(
+            ctx.accounts.token_program.to_account_info(),
+            anchor_spl::token_interface::SetAuthority {
+                current_authority: ctx.accounts.pool_authority.to_account_info(),
+                account_or_mint: ctx.accounts.base_mint.to_account_info(),
+            },
+            &[&seeds[..]],
+        ),
+        AuthorityType::MintTokens,
+        Some(Pubkey::default()),
```

Comment:
Should be `None`

Reply:
if I set to None, it is still pool authority (tested in local)

Reply:
[This](https://github.com/MeteoraAg/virtual-curve/blob/4a6e0631a19782a91ba765af794898a26c3bfb08/tests/create_pool_with_token2022.tests.ts#L162) need to be updated to `expect(baseMintData.mintAuthorityOption).eq(0);`

Reply:
Fixed, thanks